lib/anagrams.rb

```diff
@@ -1,2 +1,13 @@
 def anagrams?(str1, str2)
-end
\ No newline at end of file
+    array_1 = str1.split("")
+    array_2 = str2.split("")
+    array_3 = array_1.select { |el| !array_2.include?(el) }
+        if array_1.length == array_2.length &&
+            array_3 = array_3.empty?
```

Comment:
The conditional you seem to want is:

`If array1 and array2 are the same size and if array3 is empty`

But what you've coded would be read as:

`If array1 and array2 are the same size and if array 3 is defined as whether or not array3 is empty`

It's the `array_3 = ` part that causes the confusion. I think you would simply need `array_3.empty?` – right?